recipes/c-ares/all/conanfile.py

```diff
@@ -30,6 +37,9 @@ def _build_subfolder(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        # Default build tools to false if cross building, otherwise to true.
+        if self.options.tools == None:
+            self.options.tools = not tools.cross_building(self.settings)
```

Comment:
Not sure if we should not delegate this responsability to consumers (coherent with others recipes).

Reply:
I just think that options should have good defaults and in almost all situations I think you are not interested to have these tools for the host environment when cross building. The recipe still always you to set the option if the default is not what you want.

I'm planing to contribute similar things for other recipes (like protobuf with it's protoc tool). So I guess it good to take make a decision here and go with it :)

Reply:
Why not. Let's wait opinion of others contributors.

Anyway, you don't need `if self.options.tools == None:`

Reply:
I think the `== None` is needed. See https://github.com/conan-io/conan/issues/3519.
The "pattern" is taken from the docs: https://docs.conan.io/en/latest/reference/conanfile/attributes.html#default-options
```py
    def config_options(self):
        if self.options.some_option == None:
            if self.settings.os == 'Android':
                self.options.some_option = True
            else:
                self.options.some_option = False
```
I noted that `conan create` and `conan install` behaves differently.
With `conan install` the None check is needed but not with `conan create`. Weird..